docs: Language adjustment and typo fixes (#292)
thousandbrainsproject/tbp.monty: docs/theory/recent-progress/object-behaviors.md

# The Solution

> The Thousand Brains Theory (TBT) currently explains how an intelligent system learns the static structure of the world. In this document we describe how to extend the TBT to include behaviors. In the TBT, static objects are learned as a set of features at poses (location and orientation) in a reference frame. There is an assumption that the features are not changing or moving; when true, the existing theory and implementation works well for representing static structure.

Author: Yeah I think you're right that mine wasn't much of an improvement. Have had a go at a third description, let me know what you think.

Unlike models of static structure, models of dynamic structure consist of **sequences** of behavioral elements over time, and the timing between elements is often important. In cortical columns, we propose that cells in L4 represent features on static objects and cells in L3 represent changes in dynamic behaviors. Unlike L4 cells, L3 cells have apical dendrites that extend to L1. We propose that the apical dendrites in L1 learn the timing of events via a projection from matrix cells in the thalamus. The brain represents one implementation of this idea, but other implementations could be used (see [Implementation \- Illustrated on the Example of Monty](#implementation---illustrated-on-the-example-of-monty) for another example below).  
   
As we have previously demonstrated [2, 7, 10], cortical columns can vote to quickly reach a consensus of the static object being observed. The same mechanism applies to behaviors. Multiple columns can vote to quickly infer behaviors.  
 

### Details

> We now describe an implementation instance in greater detail, using neuroscience terms and anatomy to ground our description. However, the same principle might be implemented in other ways, just as how previously disclosed details shown in the above figure are implemented in Monty without needing to directly mirror neuroscience (see further below, [Implementation \- Illustrated on the Example of Monty](#implementation---illustrated-on-the-example-of-monty)). Although the theory maps well onto the anatomy and physiology of cortical layers as shown in the figure, the solution presented here does not depend on this mapping being correct.  

Reviewer: ```suggestion
We now describe one possible implementation in greater detail, using neuroscience terms and anatomy to ground our description. However, the same principle might be implemented in other ways, just as previously disclosed details shown in the above figure are implemented in Monty without needing to directly mirror neuroscience (see further below, [Implementation \- Illustrated on the Example of Monty](#implementation---illustrated-on-the-example-of-monty)). Although the theory maps well onto the anatomy and physiology of cortical layers as shown in the figure, the solution presented here does not depend on this mapping being correct.  
```
